swarmdjango/core/migrations/0001_initial.py

```diff
@@ -0,0 +1,183 @@
+# Generated by Django 2.2.15 on 2020-08-31 18:26
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Change',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('dateTime', models.DateTimeField(auto_now_add=True)),
+                ('context', models.TextField()),
+                ('textAdded', models.TextField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Comment',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('text', models.TextField()),
+                ('dateTime', models.DateTimeField(auto_now_add=True)),
+                ('replies', models.ManyToManyField(related_name='_comment_replies_+', to='core.Comment')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Contribution',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('link', models.TextField()),
+                ('description', models.TextField()),
+                ('fileName', models.TextField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Entry',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.TextField()),
+                ('text', models.TextField()),
+                ('comments', models.ManyToManyField(to='core.Comment')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='PersonalPage',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('pageType', models.TextField()),
+                ('pageTitle', models.TextField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='SideBar',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('content', django.contrib.postgres.fields.jsonb.JSONField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='User',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('username', models.TextField()),
+                ('password', models.TextField()),
+                ('email', models.TextField()),
```

Comment:
Do we want to have TextField here or should we use EmailField? https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.EmailField